gobblin-runtime/src/main/java/org/apache/gobblin/runtime/job_monitor/KafkaJobMonitor.java

```diff
@@ -115,4 +118,17 @@ protected void processMessage(MessageAndMetadata<byte[], byte[]> message) {
     }
   }
 
+  private void deleteStateStore(URI jobSpecUri) throws IOException {
```

Comment:
Can you add Javadoc to explain the behavior of deleteStateStore? Also, provide an example of what jobSpecUri should look like?